migration/idempotent/017-metric-rollup.sql

```diff
@@ -202,6 +223,7 @@ $$
             RAISE EXCEPTION '% rollup not found', _rollup_name;
         END IF;
         DELETE FROM _prom_catalog.metric_rollup WHERE rollup_id = _rollup_id;
+        DELETE FROM _prom_catalog.metric WHERE rollup_id = _rollup_id;
```

Comment:
>DELETE FROM _prom_catalog.metric
WHERE id IN 
(
SELECT metric_id 
FROM _prom_catalog.metric_rollup
where rollup_id = _rollup_id
);
DELETE FROM _prom_catalog.metric_rollup WHERE rollup_id = _rollup_id;

This wont work, since the row we wanna delete from metric table is not the metric_id, rather the id of the entry created by register_metric_view. The above query will delete the underlying metric from which the rollup was created, which is not the aim.